easybuild/easyconfigs/g/GATE/GATE-9.1-foss-2021b.eb

```diff
@@ -0,0 +1,28 @@
+name = 'GATE'
+version = '9.1'
+
+homepage = 'http://www.opengatecollaboration.org/'
+description = """GATE is an advanced opensource software developed by the international OpenGATE collaboration and
+ dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography
+ (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'cstd': 'c++23'}
```

Comment:
We should avoid using C++23, since it's still experimental, see https://gcc.gnu.org/projects/cxx-status.html#cxx23

Can we stick to C++17?